src/mixins/legal-api-mixin.ts

```diff
@@ -92,10 +92,12 @@ export default class LegalApiMixin extends Vue {
 
     return axios.put(url, filing).then(response => {
       const filing = response?.data?.filing
-      const filingId = +filing?.header?.filingId
+      const filingId = +filing?.header?.filingId || 0
```

Comment:
I like 0 better than undefined :)

This is also what the other methods in this class do.

Reply:
Is it to prevent a typeError and allow the method to handle an invalid response?

Reply:
No, I just prefer filingId to be a value instead of no value. I think the previous code worked just as well.